test/racingcar_test.go

```diff
@@ -0,0 +1,44 @@
+package test
+
+import (
+	"github.com/stretchr/testify/assert"
+	. "github.com/thedevluffy/go-racing-car/racingcar"
+	"testing"
+)
+
+func TestMovableCarMoveForward(t *testing.T) {
+	car := Car{Name: "개발자동차", Position: 0}
+
+	car.Move()
+
+	assert.Equal(
+		t,
+		1,
+		car.Position,
+		"Expected car position to be %d, but got %d", car.Position+1, car.Position,
+	)
+}
+
+type AlwaysTrueRule struct{}
+
+func (r AlwaysTrueRule) Check(n int) bool {
+	return true
+}
+
+func TestPlayGame(t *testing.T) {
+	alwaysTrueRule := AlwaysTrueRule{}
+
+	car1 := &Car{Name: "Car1", Position: 0}
+	car2 := &Car{Name: "Car2", Position: 0}
+	game := NewCarGame([]*Car{car1, car2}, alwaysTrueRule)
```

Comment:
rule이 함수가 될 경우 이런식으로 넘길수도 있어요.

```suggestion
func TestPlayGame(t *testing.T) {
	car1 := &Car{Name: "Car1", Position: 0}
	car2 := &Car{Name: "Car2", Position: 0}
	game := NewCarGame([]*Car{car1, car2}, func(int) bool {
		return true
	})
```